ui/app/AppLayouts/CommunitiesPortal/popups/DiscordImportProgressContents.qml

```diff
@@ -0,0 +1,313 @@
+            visible: d.status === DiscordImportProgressContents.ImportStatus.InProgress
+            font.weight: Font.Medium
+            text: qsTr("Hide window")
+            onClicked: root.close()
+        },
+        StatusButton {
+            visible: d.status === DiscordImportProgressContents.ImportStatus.CompletedSuccessfully ||
+                     d.status === DiscordImportProgressContents.ImportStatus.CompletedWithWarnings
+            font.weight: Font.Medium
+            text: qsTr("Visit your new Status community")
+            onClicked: {
+              root.close()
+                root.store.setActiveCommunity(root.store.discordImportCommunityId)
+            }
+        }
+    ]
+
+    QtObject {
+        id: d
+
+        readonly property var helperInfo: {
+            "import.communityCreation": {
+                icon: "network",
+                text: qsTr("Setting up your community")
+            },
+            "import.channelsCreation": {
+                icon: "channel",
+                text: qsTr("Importing channels")
+            },
+            "import.importMessages": {
+                icon: "receive",
+                text: qsTr("Importing messages")
+            },
+            "import.downloadAssets": {
+                icon: "image",
+                text: qsTr("Downloading assets")
+            },
+            "import.initializeCommunity": {
+                icon: "communities",
+                text: qsTr("Initializing community")
+            }
+        }
+
+        readonly property int importProgress: root.store.discordImportProgress // FIXME for now it is 0..100
+        readonly property bool importStopped: root.store.discordImportProgressStopped
+        readonly property bool hasErrors: root.store.discordImportErrorsCount
+        readonly property bool hasWarnings: root.store.discordImportWarningsCount
+
+        readonly property int status: {
+            if (importStopped) {
+                if (hasErrors)
+                    return DiscordImportProgressContents.ImportStatus.StoppedWithErrors
+                return DiscordImportProgressContents.ImportStatus.Stopped
+            }
+            if (importProgress >= 100) {
+                if (hasWarnings)
+                    return DiscordImportProgressContents.ImportStatus.CompletedWithWarnings
+                return DiscordImportProgressContents.ImportStatus.CompletedSuccessfully
+            }
+            if (importProgress > 0 && importProgress < 100)
+                return DiscordImportProgressContents.ImportStatus.InProgress
+            return DiscordImportProgressContents.ImportStatus.Unknown
+        }
+
+        function getSubtaskDescription(progress, stopped) {
+            if (progress >= 1.0)
+                return qsTr("✓ Complete")
+            if (progress > 0 && stopped)
+                return qsTr("Import stopped...")
+            if (importStopped)
+                return ""
+            if (progress <= 0.0)
```

Comment:
Shouldn't this be `>`?

Reply:
No, this means this category hasn't started progressing yet

Reply:
Ah yes, indeed, I misunderstood what Pending meant